data/successors/successor weapons.txt

```diff
@@ -0,0 +1,389 @@
+
+outfit "Drag Projectile"
+	weapon
+		sprite "projectile/drag cannon"
+			"frame rate" 20
+			"no repeat"
+		"hit effect" "drag cannon impact"
+		"inaccuracy" 20
+		"velocity" 25
+		"lifetime" 10
+		"random lifetime" 5
+		"slowing damage" 0.45
+		"shield damage" 52
+		"hull damage" 48
+
+effect "drag cannon impact"
+	sprite "effect/drag cannon impact"
+		"frame rate" 10
+		"no repeat"
+	lifetime 20
+	"velocity scale" 0
+	"random velocity" 1
+
+outfit "Successor Pulse Laser"
+	category "Guns"
+	series "Guns: Successors"
+	index 30
+	cost 69000
+	thumbnail "outfit/successor pulse laser"
+	licenses
+		"Successor"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"gun ports" -1
+	weapon
+		sprite "projectile/successor pulse laser"
+			"random start frame"
+			"frame rate" 5
+		sound "successor pulse laser"
+		"hit effect" "successor pulse laser impact"
+		"inaccuracy" 0.4
+		"velocity" 490
+		"lifetime" 1
+		"burst count" 9
+		"burst reload" 1
+		"reload" 6
+		"firing energy" 11
+		"firing heat" 15
+		"shield damage" 10.5
+		"hull damage" 12
+		"piercing" 0.05
+	description "This shield-penetrating laser is one of the few weapons manufactured independently in Successor space, rather than under the auspices and restrictions of the High Houses. As a consequence of this, it is neither especially powerful nor efficient, but its low cost and space requirements make it a nonetheless relatively appealing defensive choice for those without access to the High Houses' more potent weaponry."
+
+outfit "Pulse Laser Turret"
+	category "Turrets"
+	series "Turrets: Successors"
+	index 20
+	cost 145000
+	thumbnail "outfit/successor pulse laser turret"
+	licenses
+		"Successor"
+	"mass" 30
+	"outfit space" -30
+	"weapon capacity" -30
+	"turret mounts" -1
+	weapon
+		sprite "projectile/successor pulse laser"
+			"random start frame"
+			"frame rate" 5
+		"hardpoint sprite" "hardpoint/successor pulse laser turret"
+		"hardpoint offset" 8
+		sound "successor pulse laser"
+		"hit effect" "successor pulse laser impact"
+		"inaccuracy" 0.4
+		"velocity" 490
+		"turret turn" 2.5
+		"lifetime" 1
+		"burst count" 9
+		"burst reload" 1
+		"reload" 3
+		"firing energy" 11
+		"firing heat" 15
+		"shield damage" 10.5
+		"hull damage" 12
+		"piercing" 0.05
+	description "The introduction of the Pulse Laser was a wake-up call for the more astute in the High Houses, as it was the first time in hundreds of years that weapons were being manufactured by groups other than themselves. Most Successors see the turreted version as a continuation of this democratization of violence, while the more paranoid claim that the High Houses have infiltrated production and intentionally sabotaged the design."
+
+effect "successor pulse laser impact"
+	sprite "effect/successor pulse laser impact"
+		"frame rate" 30
+		"random start frame"
+	lifetime 3
+	"velocity scale" 0
+
+outfit "Overcharged Pulse Laser"
```

Comment:
I would consider this to be a noticeably weaker outfit than the standard pulse laser. Its dps/ton is slightly worse, it's slightly easier to support, and having higher shield damage but lower hull damage makes the piercing less useful. The firing cycle is too short to really take advantage of the burst fire with cluster firing mode, too.